openslides/motions/projector.py

```diff
@@ -9,24 +9,91 @@
 #            to be fast!
 
 
-def motion(
-    element: Dict[str, Any], all_data: Dict[str, Dict[int, Dict[str, Any]]]
+def get_state(
+    all_data: AllData, motion: Dict[str, Any], state_id: int
 ) -> Dict[str, Any]:
+    """
+    Returns a state element from one motion.
+
+    Returns an error if the state_id does not exist for the workflow in the motion.
+    """
+    states = all_data["motions/workflow"][motion["workflow_id"]]["states"]
+    for state in states:
+        if state["id"] == state_id:
+            return state
+    return {"error": f"motion {motion['id']} can not have to id {state_id}"}
+
+
+def motion_slide(element: Dict[str, Any], all_data: AllData) -> Dict[str, Any]:
     """
     Motion slide.
```

Comment:
Please extend docstring and describe what pieces of information may be one the motion slide.

Reply:
Then I have to write all python code once more in english. The function does nothing else then describe what pieces of information is on the motion slide